File: Problem Set 1 Mario/mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_int(void);

int main(void)
{
    // user input
    int height = get_positive_int();

    for(int i=height;i > 0;i--)
    {
       for(int j=0;j < i-1;j++)
       {
           printf(" ");

       }
       for(int k=0;k < (height+1-i);k++)
       {
           printf("#");
       }
        printf("  ");
        for(int m=0;m<height-i+1;m++)
        {
            printf("#");
        }
            printf("\n");
    }
}
int get_positive_int(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while(n < 1 || n > 8);
    return n;
}

File: Problem Set 6/mario.py
from cs50 import get_int

while True:
    height = get_int("height: ")
    if height < 9 and height > 0:
        break


count = 1
for h in range(height):
    print(" " * (height-count), end="")
    print("#" * count)
    count += 1
